### ISSUE TITLE
Checkbox: accessibility fix

### ISSUE THREAD

**Issue by PosikBoy** [gravity-ui/uikit#2502]
### Describe the bug

If the checkbox receives a default checked state (i.e., true), then after the user unchecks it, the component produces an inconsistent state between the DOM and the aria-checked attribute.

The Checkbox component adds a redundant aria-checked attribute to the native <input type="checkbox">, which:
- Causes accessibility test failures (aria-conditional-attr)
- Is unnecessary, because browsers already expose the checked state of native checkboxes in the accessibility tree
- Conflicts with the WAI-ARIA specification

### Reproduction

https://stackblitz.com/edit/gravity-ui-uikit-example-g5ht5ume?file=src%2FApp.tsx

### Steps to reproduce

Uncheck checkbox and open dev tools. You see inconsistent state with aria attribute and checked

### Environment

macos, yandex browser

### Validations

- [x] Check that there isn't [already an issue](https://github.com/gravity-ui/uikit/issues) that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ValeraS** [comment]
[checked](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/checkbox#checked) attribute indicating whether this checkbox is checked by default and does not indicate whether this checkbox is currently checked.

`Checkbox` component supports tri-state checkbox. For this purpose it uses `indeterminate` state and `aria-checked="mixed"`. `aria-checked` attribute is synchronised with `checked` and `indeterminate` states. So reproduction is incorrect, you need to compare internal `checked` state with `aria-checked` attribute.

**Comment by PosikBoy** [comment]
I will make pull request for this issue
